pkgs/tools/admin/yandexcloud-cli/default.nix

```diff
@@ -0,0 +1,121 @@
+{ lib
+, stdenv
+, fetchurl
+, makeBinaryWrapper
+, installShellFiles
+, testers
+  # update script
+, writeShellScript
+, curl
+, common-updater-scripts
+}:
+let
+  version = "0.107.0";
+  hashes = {
+    # Linux
+    "aarch64-linux" = "sha256-KCvztarM9Y3ul+dQD8yscZ9or79tChi6SOvv7SQc0w8=";
+    "x86_64-linux" = "sha256-DGueeBc2tFm94cI+TlbjFVxCCrGljaJGIFInXt9Dnp0=";
+    "i686-linux" = "sha256-5P7zu1P6SBazJV5DVEVim6mqhFOW0FNXy3MUhjqZ5eA=";
+    # Darwin
+    "aarch64-darwin" = "sha256-NYj/9rhMglDtgnQVKRAL96q0SXlDzd+3km5G2mpctIE=";
+    "x86_64-darwin" = "sha256-J7rAxUEw50bsWb9lCxKWraH+d1WnH/fWcsOQ1RkuJ0s=";
+    # Windows
+    "x86_64-windows" = "sha256-aT+FYc3FD8L6KSD3pNvvSbUkNCAyofkfjGBmGN96OS0=";
+    "i686-windows" = "sha256-wqSUspHcMqFKaXILK7PdnnnoWSLycL+4HDXXya2iTCI=";
+  };
+
+  storageURL = "https://storage.yandexcloud.net/yandexcloud-yc/release";
+
+  fetchSource = system: hash:
+    let
+      throwSystem = throw "Unsupported system: ${system}";
+      platform = lib.systems.elaborate system;
+      arch =
+        if platform.isAarch64 then "arm64"
+        else if platform.isx86_64 then "amd64"
+        else if platform.isx86_32 then "386"
+        else throwSystem;
+      os =
+        if platform.isLinux then "linux"
+        else if platform.isDarwin then "darwin"
+        else if platform.isWindows then "windows"
+        else throwSystem;
+      bin = "yc${lib.optionalString platform.isWindows ".exe"}";
```

Comment:
I am not sure if windows support even would work in nixpkgs

Reply:
Well, it works, `NIXPKGS_ALLOW_UNFREE=1 nix-build -A pkgsCross.mingwW64.yandexcloud-cli`

But yeah, that probably doesn’t make much sense, removed it.